src/elm/Page/Dashboard/Claim.elm

```diff
@@ -3,7 +3,7 @@ module Page.Dashboard.Claim exposing (Model, Msg, init, jsAddressToMsg, msgToStr
 import Action
 import Api.Graphql
 import Cambiatus.Query
-import Claim
+import Claim exposing (VotingProgressBar(..), viewVotingProgress)
```

Comment:
IMO it's usually clearer to use `Claim.ClaimModalAndDetailPage` and `Claim.viewVotingProgress` (instead of exposing them). We have a few modules where we use exposed functions (mainly the `Html` ones), but I generally prefer qualified names. (You don't need to change this)

Reply:
Done.